src/components/search/concept-modal/concept-modal.js

```diff
@@ -98,12 +99,78 @@ export const ConceptModalBody = ({ result }) => {
   )
 
   const tabs = {
-    'overview':     { title: 'Overview',         icon: <OverviewIcon />,         content: <OverviewTab result={ result } />, },
-    'studies':      { title: studyTitle,         icon: <StudiesIcon />,          content: <StudiesTab studies={ studies } />, },
-    'cdes':         { title: cdeTitle,           icon: <CdesIcon />,             content: <CdesTab cdes={ cdes } cdeRelatedConcepts={ cdeRelatedConcepts } loading={ cdesLoading } /> },
-    'kgs':          { title: 'Knowledge Graphs', icon: <KnowledgeGraphsIcon />,  content: <KnowledgeGraphsTab graphs={ graphs } />, },
-    'explanation':  { title: 'Explanation',      icon: <ExplanationIcon />,      content: <ExplanationTab result={ result } /> },
-    'tranql':       { title: 'TranQL',           icon: <TranQLIcon />,           content: <TranQLTab result={ result } graphs = { graphs } /> }
+    'overview': {
+      title: 'Overview',
+      icon: <OverviewIcon />,
+      content: <OverviewTab result={ result } />,
+      tooltip: <div>
+        The Overview tab provides a brief description of the concept.
+        Hovering over the <InfoCircleOutlined style={{ color: "rgba(0, 0, 0, 0.45)", margin: "0 2px" }} />
```

Comment:
This is a little unclear, because the InfoCircleOutlined isn't actually in the Overview tab but _above_ the Overview tab (as part of the concept model's title).

Reply:
true, not sure this belongs here. I think it's more so filler because there's not much to say about an overview?